base/applications/regedit/framewnd.c

```diff
@@ -881,12 +891,7 @@ static INT_PTR CALLBACK AddToFavoritesDlgProc(HWND hWnd, UINT uMsg, WPARAM wPara
                         err = RegSetValueExW(hKey, name, 0, REG_SZ, (BYTE*)path, (lstrlenW(path) + 1) * sizeof(WCHAR));
                         RegCloseKey(hKey);
                         if (err) failed:
-                        {
-                            if (!FormatMessageW(FORMAT_MESSAGE_FROM_SYSTEM | FORMAT_MESSAGE_IGNORE_INSERTS,
-                                                NULL, err, 0, name, _countof(name), NULL))
-                                *(UINT*)name = L'?';
-                            MessageBoxW(hWnd, name, NULL, MB_ICONSTOP);
-                        }
+                            ErrorBox(hWnd, err);
                         if (path)
```

Comment:
https://stackoverflow.com/questions/1912325/checking-for-null-before-calling-free

Reply:
Ah yes, because I used GlobalFree first and that used to crash on NULL.